.github/workflows/actions/publish-npm-package/action.yml

```diff
@@ -81,3 +81,17 @@ runs:
       npm whoami
       cd ${{ inputs.package }}/pkg
       npm publish --tag ${{ inputs.tag }} --access ${{ inputs.access }}
+
+  - name: Promote package distribution tag to 'latest'
+    if: inputs.dry_run == 'false' && steps.check_version.outputs.should_deploy == 'true' && inputs.tag == 'latest'
+    shell: bash
+    env:
+      NPM_TOKEN: ${{ inputs.api_token }}
+    run: |
+      echo "Publish '@${{ inputs.scope }}/${{ inputs.package }}' package"
+      npm set "//registry.npmjs.org/:_authToken=${NPM_TOKEN}"
+      npm whoami
+      cd ${{ inputs.package }}/pkg
+      PACKAGE_VERSION=$(npm pkg get version | tr -d '"')
```

Comment:
It looks like the package version has already been computed above with `LOCAL_VERSION`